### ISSUE TITLE
Hide mouse cursor

### ISSUE THREAD

**Issue by crazyyzarc** [edrlab/thorium-reader#1505]
Could you add a function to hide automatically mouse cursor?

**Comment by danielweck** [comment]
Fixed via https://github.com/edrlab/thorium-reader/commit/bda8f0ae218fd8fbdde4aca1ad9c454b91c73e41 and https://github.com/readium/r2-navigator-js/commit/fd1bb4b4b4908fc821ecd18af011f4de20f73e54

Note that the publication document viewport and Thorium's user interface controls have their own interaction "boundaries" which do not overlap. In layman's terms: moving the mouse cursor inside the keyboard-focused publication document viewport does not trigger mouse move events inside Thorium's UI (in other words: EPUB XHTML documents "capture" mouse and keyboard events). Based on my own tests this does not seem to cause any confusion, but please feel free to report back here with your own feedback :)
